src/components/Modal.js

```diff
@@ -0,0 +1,59 @@
+import styled from 'styled-components'
+
+const Modal = ({ state, statechange }) => {
+  return (
+    <>
+      {state && (
+        <Overlay>
+          <Container>
+            <ModalInfo>Insert Modal Message Here</ModalInfo>
+            <AcceptButton onClick={() => statechange(false)}>Insert Button Text Here</AcceptButton>
+          </Container>
+        </Overlay>
+      )}
+    </>
+  )
+}
+
+export default Modal
+
+const Overlay = styled.div`
+  width: 100vw;
+  height: 100vh;
+  position: fixed;
+  top: 0;
+  left: 0;
+  background: rgba(255, 255, 255, 0.4);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`
+
+const Container = styled.div`
+  width: 500px;
+  min-height: 360px;
+  background: #fff;
```

Comment:
please use css variable, for example:
```suggestion
  background: var(--dasher-white);
```

Reply:
Corrected